### PR TITLE
Diane

### PR DESCRIPTION
**Pull request by dtpratte**
Not sure this is right.

### CODE REVIEW

File: _pages/contact-us.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ redirect_from:
 **If you have an Eligibility Application Pending  - Call**  
 ***Joseph Shelton  extension  #151   [jshelto@onhir.gov](mailto:jshelto@onhir.gov)***  
 ***Bertha Begay     extension  #142   [bbegay@onhir.gov](mailto:bbegay@onhir.gov)***  
-***Colleen Tso        extension  #116   [ctso@onhir.gov](mailto:ctso@onhir.gov)***
+*
```
Comment (amirbey) [inline _pages/contact-us.md]: is this asterisk intentional?

File: _pages/contact-us.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ TTY 1-800-877-8339
 
 **REMOTE OFFICE - Sanders, AZ**  
 **Mae Salago-King     **[mking@onhir.gov](mailto:mking@onhir.gov?subject=Contact%20from%20ONHIR%20Website)  
-**Helen Benally           **[hbennall@onhir](mailto:mking@onhir.gov?subject=Contact%20from%20ONHIR%20Website)
+*
```
Comment (amirbey) [inline _pages/contact-us.md]: is this asterisk intentional?

**Comment by dtpratte** [comment]
yes that is intentional -


I'm worried about the work I've done in the last number of days.  I started working in the Diane fork and later realized I was in the master.  Will the merges work?

Also, I know I have some more linking problems so after the merge I'll need to bring all down to the Diane fork to start over checking all my links.   How do I clean out what is in Diane and bring down the full master?


Otherwise, I think I'm doing all my fixes correctly.   There seems to be a problem with the `_config.yml` file on line 27, not sure if I fixed my problem or not.


Diane


Diane Pratte
Chief Information Officer,
Office of Navajo and Hopi Indian Relocation
PO Box KK, Flagstaff, AZ  86002
(928) 779-2727 ext. 139
(928) 774-1977 (fax)

________________________________

**Comment by amirbey** [comment]
I have removed your the offending text from the config.yml and the site now builds successfully in Federalist.  You can preview your site on Federalist to make sure you feel good about all your changes.  It looks like you've been making changes to both the diane and master branch. be sure to pull my commits down which have fixed the config.yml error in both diane and mster branch

File: _pages/contact-us.md (hunk 1 above)
Comment (dtpratte) [reply _pages/contact-us.md]: I removed someones name and just left the ** for looks other than a blank line.

**Comment by amirbey** [comment]
hi @dtpratte  - this PR is getting pretty big. are you going to merge this PR soon?

**Comment by dtpratte** [comment]
What is PR?   and going to merge PR soon?


Diane

Diane Pratte
Chief Information Officer,
Office of Navajo and Hopi Indian Relocation
PO Box KK, Flagstaff, AZ  86002
(928) 779-2727 ext. 139
(928) 774-1977 (fax)

________________________________

**Comment by amirbey** [comment]
PR => pull request

**Pull request merged**